### DISCUSSION TITLE
Is MantineColor missing dimmed color value?

### DISCUSSION THREAD

**Discussion by sl-ishvedetskyi** [mantinedev/mantine#1368] [Q&A] [answered]
Hello!

Is MantineColor missing dimmed value? When selecting color value for some components, i.e. Text component, there is no 'dimmed' value. For some reason, I expected it to be there.

<img width="690" alt="Screen Shot 2022-05-02 at 4 06 49 PM" src="https://user-images.githubusercontent.com/98319155/166238723-4c8857ec-9c54-4e8f-afb7-556d127569d8.png">

**Comment by rtivital** [comment, marked as answer]
No MantineColor is not missing `dimmed` value, it is unique to Text component.

**Reply by zhulien-ivanov** [reply]
Maybe we can change `Text`'s `color` property to a union: `color?: "dimmed" | MantineColor;`. This way we can get correct intellisense for `dimmed` as well.

**Reply by rtivital** [reply]
Yep, that should work

**Reply by zhulien-ivanov** [reply]
I'll add a PR.
